CONTRIBUTING.md

```diff
@@ -23,3 +23,33 @@ If you'd like to help make RSpec better, here are some ways you can contribute:
 If you need help getting started, check out the [DEVELOPMENT](DEVELOPMENT.md) file for steps that will get you up and running.
 
 Thanks for helping us make RSpec better!
+
+## Rspec issues labels definition
+
+### `Your first PR` issues
+
+These issues are the ones that we be believe are best suited for new
+contributors to get started on. They represent a potential meaningful
+contribution to the project that should not be too hard to pull off.
+
+### `Needs reproduction case` issues
+
+These issues are ones that have been labelled by the maintainers that we
+believe do not currently have enough information to be reproduced the RSpec
+team. While not directly counted by the GitHub contribution graph, we consider
+helping us to reproduced the issue with a repro case as an extremely meaningful
```

Comment:
`helping us to reproduced the issue with a repro case` would be replaced by`giving/offering the reproduction steps`

Reply:
Again the english is correct here, apart from a typo, `s/reproduced/reproduce`.

Reply:
@benoittgt can we get the typo here fixed up?